src/shared/components/atoms/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,61 @@
+import classNames from "classnames";
+import * as React from "react";
+import { Icon } from "../Icon";
+
+import "./Avatar.scss";
+
+interface Props {
+    avatarInitials?: string;
+    avatarName?: string;
+    avatarURL?: string;
+    className?: string;
+}
+
+class Avatar extends React.Component<Props> {
+    public render() {
+        const {
+            avatarURL,
+            avatarInitials,
+            avatarName,
+            className,
+        } = this.props;
+        const avatarClasses = classNames(className, {
+            "a-avatar": true,
+            "a-avatar--anonymous": (!avatarURL || !avatarName) && !avatarInitials,
+            "a-avatar--initials": !avatarURL && avatarInitials,
+        });
+        // TODO Update the url pattern to match the image service
+        const srcSet = `
+            ${avatarURL}/38,
+            ${avatarURL}/76 2x,
+            ${avatarURL}/114 3x,
+            ${avatarURL}/152 4x
+        `;
+
+        if (avatarURL && avatarName) {
+            return(
+                <img
+                    alt={avatarName}
```

Comment:
Could we make this truly optional, so if the name is next to the avatar, we can have an empty alt?

Reply:
Good call, updated.